mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -667,3 +673,29 @@ case class ExtensionParamOccurence(
     sym: Symbol,
     methods: List[untpd.Tree],
 )
+
+case class EndMarker(symbol: Symbol)
+
+object EndMarker:
+  val endMarkerRegex = """.*end(/\*.*\*/|\s)+""".r
+  def getPosition(df: NamedDefTree, pos: SourcePosition, sourceText: String)(
+      implicit ct: Context
+  ): Option[SourcePosition] =
+    val name = df.name.toString()
+    val endMarkerLine =
+      sourceText.slice(df.span.start, df.span.end).split('\n').last
```

Comment:
This has popped up recently a few times already: is just `\n` enough, or would it make sense to check other newline combinations as well? :slightly_smiling_face:

Reply:
Shouldn't matter (unless someone has random carriage returns in code).